_drafts/type-inference-part-2.md

```diff
@@ -8,17 +8,17 @@ Previously on part 1 we saw the very basics of how the type inference algorithm
 
 The next step is to introduce functions. _defs_ in crystal don't need to have type annotations. So a programmer could write the following:
 
-<div class="code_section">{% highlight ruby %}
+```ruby
```

Comment:
Sounds good. But I think this is a separate change. 
I presume `ruby` was historically selected for technical reasons, due to no specific highlighter being available. But we've had highlighter support for Crystal syntax for a long time, yet these old code blocks still use Ruby.
In the end it's also probably not very important to change this.

Reply:
As you wish, but to me this is like the right moment to do the change, given that we're looking at it. Exactly because it's not an important change, it's the kind of thing that we'll might not create a PR for in the future.